app/js/components/LabOrdersList/LabOrdersList.jsx

```diff
@@ -7,13 +7,131 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 import React from 'react';
+import { connect } from 'react-redux';
+import cn from 'classnames';
+import moment from 'moment';
+import Loading from '../Loading';
+import SortableTable from '../SortableTable';
+import { fetchLabOrders } from '../../actions/labOrderAction';
+import "react-table/react-table.css";
+import "./index.scss";
+
+
+
+const Cell = ({columnName, value}) => {
+  if (columnName === "EMR ID") {
+    const emrID = value.patient.display.split('-')[0];
+    return (
+      <div className= "table-cell emr-id">
+        <span>{emrID}</span>
+      </div>
+    );
+  }
+  if (columnName === "NAME") {
+    const displayName = value.patient.display.split('-')[1];
+    return (
+      <div className= "table-cell order-date">
+        <span>{displayName}</span>
+      </div>
+    );
+  }
+  if (columnName === "ORDER ID") {
+    return (
+      <div className= "table-cell order-id">
+        <span>{value.orderNumber}</span>
+      </div>
+    );
+  }
+  if (columnName === "ORDER DATE") {
+    return (
+      <div className= "table-cell order-date">
+        <span>{moment(value.dateActivated).format("D-MMM-YYYY")}</span>
```

Comment:
Can we pull the date format out into a constant so it's easier to change if needed?

Reply:
yeah, that makes sense